eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -32,6 +32,9 @@ parameters:
   - name: TestEnv
     type: object
     default: []
+  - name: VcpkgCacheMode
+    type: string
+    default: 'read'    
```

Comment:
from where ?

Reply:
binary cache mode has 2 operation modes, `read` and `readwrite`.

read will only try to fetch and restore binaries from the cache, while readwrite will push binaries to the cache after building them.